Kickstarter-iOS/Views/Controllers/PledgeAmountViewControllerTests.swift

```diff
@@ -0,0 +1,139 @@
+@testable import Kickstarter_Framework
+@testable import Library
+@testable import KsApi
+import Prelude
+import UIKit
+
+final class PledgeAmountViewControllerTests: TestCase {
```

Comment:
I felt like it was a perfect opportunity to add testing...not only can we test large text but also invalid input values, currency symbol, stepper increment/decrement button etc.